cloudbridge/cloud/providers/gce/resources.py

```diff
@@ -912,19 +912,191 @@ def create_image(self, name):
         raise NotImplementedError(
             'To be implemented after GCEVolumeService.')
 
+    @property
+    def _existing_target_instance(self):
+        self_url = self._provider.parse_url(self._gce_instance['selfLink'])
+        try:
+            response = (self._provider.gce_compute
+                                      .targetInstances()
+                                      .list(project=self._provider.project_name,
```

Comment:
Yes, it is a part of the selfLink. But, is there a reason to not use `self._provider.project_name`? It is consistent with all other places where we use the `list()` method, and less importantly, it is 3 characters shorter than `self_url.parameters['project']`, which makes it fit in the same line :).